src/openshift/application.ts

```diff
@@ -24,17 +24,24 @@ export class Application extends OpenShiftItem {
         return null;
     }
 
-    static async getApplicationName() {
+    static async getApplicationName(project) {
         return await vscode.window.showInputBox({
             prompt: "Application name",
-            validateInput: (value: string) => {
-                const validationMessage = Application.emptyName('Empty application name', value.trim());
-                if (!validationMessage) return Application.validateMatches('Not a valid Application name. Please use lower case alphanumeric characters or "-", and must start and end with an alphanumeric character', value);
+            validateInput: async (value: string) => {
+                let validationMessage = Application.emptyName('Empty application name', value.trim());
+                if (!validationMessage) validationMessage = Application.validateMatches('Not a valid Application name. Please use lower case alphanumeric characters or "-", and must start and end with an alphanumeric character', value);
+                if (!validationMessage) validationMessage = Application.lengthName('Storage name is to long', value);
```

Comment:
You mean 'Application name is too long'?

Reply:
Yes thanks @jrichter1